src/LocalCoverage.jl

```diff
@@ -195,8 +190,7 @@ end
 """
 $(SIGNATURES)
 
-Generate, and optionally open, the HTML coverage summary in a browser for `pkg`
-inside `dir`.
+Open the HTML coverage results in a browser for `pkg` if they exist.
```

Comment:
I think juet that we could keep the information about the other parameters in this doc.
```suggestion
Generate, and optionally open, the HTML coverage summary in a browser for `pkg`
inside `dir`.
```